installation problem

When I try to instal node-sass it shows up :
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "install" "node-sass"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4

npm ERR! Cannot read property 'trim' of undefined
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Souhaib\npm-debug.log

I followed the steps but I still get the same error :
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "install" "node-sass"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4

npm ERR! Cannot read property 'trim' of undefined
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
-
npm ERR! Please include the following file with any support request:
npm ERR!     C:\temp1\npm-debug.log

C:\temp1>cd node_modules\node-sass\vendor
The system cannot find the path specified.

--- a/package.json
+++ b/package.json
@@ -57,7 +57,7 @@
     "mkdirp": "^0.5.0",
     "nan": "^1.8.4",
     "npmconf": "^2.1.1",
-    "pangyp": "am11/pangyp",
+    "pangyp": "https://github.com/am11/pangyp/archive/f13599f5198b853948429062c2b334010a337342.tar.gz",
     "request": "^2.55.0",
     "sass-graph": "^2.0.0"
   },

